entropylab/results/dashboard/dashboard.py

```diff
@@ -200,4 +219,16 @@ def activate_last_plot_tab_when_tabs_are_changed(children):
             return children[last_tab]["props"]["tab_id"]
         return 0
 
+    @_app.callback(
+        Output("aggregate-clipboard", "content"),
+        Input("aggregate-clipboard", "n_clicks"),
+        State("aggregate-graph", "figure"),
+        prevent_initial_call=True,
+    )
+    def copy_aggregate_data_to_clipboard_as_python_code(_, figure):
+        _dict = list(
+            map(lambda d: {d["name"]: {"x": d["x"], "y": d["y"]}}, figure["data"])
+        )
+        return f"data = {_dict}"
```

Comment:
@urig 
I think that we need to use a better serialization here. the data will probably be only numeric so the current solution will work. but maybe using pickle will be more safe?

Reply:
also, please add a test for the function and make sure loading it in works

Reply:
@qguyk thanks for the feedback. Re: pickling, let's ask @galwiner . The idea I got from him was to keep things nice and simple.

Reply:
we can give a string 
'pickle.loads("actual_pickled_data")'
and then running it in the python console results with a valid dictionary.

I agree, we need @galwiner input here

Reply:
I suggest we start without pickling. It's all going to be arrays as it's coming from a plot or set of plots. 
If users want to do something more involved they can use the DB API to collect the data they need.
This gives me an idea for an enhancement and I will open an issue. (but it's not for now)

Reply:
actually I think that numpy would not work this way. numpy has built in toString() and fromString() functions

Reply:
I don't understand. 
If there are multiple arrays to return, then we can return multiple (numpy) arrays. 
one for the x coordinates, one for the y coordinates and a 2-d (or more -d) array containing the data

Am I missing something?

Reply:
@galwiner @qguyk the way I implemented it, the code is generated from the plotly figure. I will check but I'm pretty sure that always contains a plain array implementation of the data, even if it came from an `ndarray`. Is this ok or should I get the data directly from the `PlotRecord` entities (which are sometimes `ndarray`)?

Reply:
I checked: ndarrays are passed through the plotly figures and the code pastes them correctly to the clipboard (see unit tests added)

Reply:
Resolved by outputing `np.array`.